components/sup/src/manager/file_watcher.rs

```diff
@@ -2402,15 +2379,40 @@ mod tests {
                                            events: vec![NotifyEvent::disappeared(pb!("/a/b/c"))], },], },]
     }
 
+    fn run_test_case(tc: &TestCase) {
+        let mut runner = TestCaseRunner::new();
+        runner.debug_info.add(format!("test case: {}", tc.name));
+        runner.run_init_commands(&tc.init.commands);
+        let setup = runner.prepare_watcher(&tc.init.path);
+        runner.run_steps(setup, &tc.init.initial_file, &tc.steps);
+    }
+
     #[test]
     fn file_watcher() {
+        let lock = lock_env_var();
+        lock.set("");
```

Comment:
this could just be `lock.unset()` which will clear out the variable.

Reply:
OK - there are a number of other instances as well across our test cases, so will fix all of them.